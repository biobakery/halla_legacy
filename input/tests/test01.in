FID	C1	C2	C3	C4	C5	C6
D1	0	0	0	1	1	1
D2	A	A	B	B	C	C
D3	0.1	0.2	0.3	1	2	3
D4	1	2	3	4	5	6
