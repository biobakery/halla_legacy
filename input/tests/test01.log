2012-10-13 12:54:10,943       INFO dataset.permute@90 Generating 41 permutations
2012-10-13 12:54:10,943       INFO dataset.permute@94 Permutation 0/41
2012-10-13 12:54:10,944       INFO dataset.permute@94 Permutation 1/41
2012-10-13 12:54:10,944       INFO dataset.permute@94 Permutation 2/41
2012-10-13 12:54:10,944       INFO dataset.permute@94 Permutation 3/41
2012-10-13 12:54:10,945       INFO dataset.permute@94 Permutation 4/41
2012-10-13 12:54:10,945       INFO dataset.permute@94 Permutation 5/41
2012-10-13 12:54:10,945       INFO dataset.permute@94 Permutation 6/41
2012-10-13 12:54:10,946       INFO dataset.permute@94 Permutation 7/41
2012-10-13 12:54:10,946       INFO dataset.permute@94 Permutation 8/41
2012-10-13 12:54:10,946       INFO dataset.permute@94 Permutation 9/41
2012-10-13 12:54:10,947       INFO dataset.permute@94 Permutation 10/41
2012-10-13 12:54:10,947       INFO dataset.permute@94 Permutation 11/41
2012-10-13 12:54:10,947       INFO dataset.permute@94 Permutation 12/41
2012-10-13 12:54:10,948       INFO dataset.permute@94 Permutation 13/41
2012-10-13 12:54:10,948       INFO dataset.permute@94 Permutation 14/41
2012-10-13 12:54:10,948       INFO dataset.permute@94 Permutation 15/41
2012-10-13 12:54:10,948       INFO dataset.permute@94 Permutation 16/41
2012-10-13 12:54:10,949       INFO dataset.permute@94 Permutation 17/41
2012-10-13 12:54:10,949       INFO dataset.permute@94 Permutation 18/41
2012-10-13 12:54:10,949       INFO dataset.permute@94 Permutation 19/41
2012-10-13 12:54:10,950       INFO dataset.permute@94 Permutation 20/41
2012-10-13 12:54:10,950       INFO dataset.permute@94 Permutation 21/41
2012-10-13 12:54:10,950       INFO dataset.permute@94 Permutation 22/41
2012-10-13 12:54:10,951       INFO dataset.permute@94 Permutation 23/41
2012-10-13 12:54:10,951       INFO dataset.permute@94 Permutation 24/41
2012-10-13 12:54:10,951       INFO dataset.permute@94 Permutation 25/41
2012-10-13 12:54:10,952       INFO dataset.permute@94 Permutation 26/41
2012-10-13 12:54:10,952       INFO dataset.permute@94 Permutation 27/41
2012-10-13 12:54:10,952       INFO dataset.permute@94 Permutation 28/41
2012-10-13 12:54:10,953       INFO dataset.permute@94 Permutation 29/41
2012-10-13 12:54:10,953       INFO dataset.permute@94 Permutation 30/41
2012-10-13 12:54:10,953       INFO dataset.permute@94 Permutation 31/41
2012-10-13 12:54:10,953       INFO dataset.permute@94 Permutation 32/41
2012-10-13 12:54:10,954       INFO dataset.permute@94 Permutation 33/41
2012-10-13 12:54:10,954       INFO dataset.permute@94 Permutation 34/41
2012-10-13 12:54:10,954       INFO dataset.permute@94 Permutation 35/41
2012-10-13 12:54:10,955       INFO dataset.permute@94 Permutation 36/41
2012-10-13 12:54:10,955       INFO dataset.permute@94 Permutation 37/41
2012-10-13 12:54:10,955       INFO dataset.permute@94 Permutation 38/41
2012-10-13 12:54:10,956       INFO dataset.permute@94 Permutation 39/41
2012-10-13 12:54:10,956       INFO dataset.permute@94 Permutation 40/41
2012-10-13 12:54:10,957       INFO dataset.hierarchy@205 Cluster 1 (3): ['D1', 'D3', 'D4']
2012-10-13 12:54:10,957       INFO dataset.bootstrap@104 Generating 100 bootstraps
2012-10-13 12:54:10,957       INFO dataset.bootstrap@107 Bootstrap 0/100
2012-10-13 12:54:10,957       INFO dataset.bootstrap@107 Bootstrap 1/100
2012-10-13 12:54:10,957       INFO dataset.bootstrap@107 Bootstrap 2/100
2012-10-13 12:54:10,958       INFO dataset.bootstrap@107 Bootstrap 3/100
2012-10-13 12:54:10,958       INFO dataset.bootstrap@107 Bootstrap 4/100
2012-10-13 12:54:10,958       INFO dataset.bootstrap@107 Bootstrap 5/100
2012-10-13 12:54:10,959       INFO dataset.bootstrap@107 Bootstrap 6/100
2012-10-13 12:54:10,959       INFO dataset.bootstrap@107 Bootstrap 7/100
2012-10-13 12:54:10,959       INFO dataset.bootstrap@107 Bootstrap 8/100
2012-10-13 12:54:10,959       INFO dataset.bootstrap@107 Bootstrap 9/100
2012-10-13 12:54:10,960       INFO dataset.bootstrap@107 Bootstrap 10/100
2012-10-13 12:54:10,960       INFO dataset.bootstrap@107 Bootstrap 11/100
2012-10-13 12:54:10,960       INFO dataset.bootstrap@107 Bootstrap 12/100
2012-10-13 12:54:10,960       INFO dataset.bootstrap@107 Bootstrap 13/100
2012-10-13 12:54:10,961       INFO dataset.bootstrap@107 Bootstrap 14/100
2012-10-13 12:54:10,961       INFO dataset.bootstrap@107 Bootstrap 15/100
2012-10-13 12:54:10,961       INFO dataset.bootstrap@107 Bootstrap 16/100
2012-10-13 12:54:10,962       INFO dataset.bootstrap@107 Bootstrap 17/100
2012-10-13 12:54:10,962       INFO dataset.bootstrap@107 Bootstrap 18/100
2012-10-13 12:54:10,962       INFO dataset.bootstrap@107 Bootstrap 19/100
2012-10-13 12:54:10,962       INFO dataset.bootstrap@107 Bootstrap 20/100
2012-10-13 12:54:10,963       INFO dataset.bootstrap@107 Bootstrap 21/100
2012-10-13 12:54:10,963       INFO dataset.bootstrap@107 Bootstrap 22/100
2012-10-13 12:54:10,963       INFO dataset.bootstrap@107 Bootstrap 23/100
2012-10-13 12:54:10,963       INFO dataset.bootstrap@107 Bootstrap 24/100
2012-10-13 12:54:10,964       INFO dataset.bootstrap@107 Bootstrap 25/100
2012-10-13 12:54:10,964       INFO dataset.bootstrap@107 Bootstrap 26/100
2012-10-13 12:54:10,964       INFO dataset.bootstrap@107 Bootstrap 27/100
2012-10-13 12:54:10,964       INFO dataset.bootstrap@107 Bootstrap 28/100
2012-10-13 12:54:10,965       INFO dataset.bootstrap@107 Bootstrap 29/100
2012-10-13 12:54:10,965       INFO dataset.bootstrap@107 Bootstrap 30/100
2012-10-13 12:54:10,965       INFO dataset.bootstrap@107 Bootstrap 31/100
2012-10-13 12:54:10,966       INFO dataset.bootstrap@107 Bootstrap 32/100
2012-10-13 12:54:10,966       INFO dataset.bootstrap@107 Bootstrap 33/100
2012-10-13 12:54:10,966       INFO dataset.bootstrap@107 Bootstrap 34/100
2012-10-13 12:54:10,966       INFO dataset.bootstrap@107 Bootstrap 35/100
2012-10-13 12:54:10,967       INFO dataset.bootstrap@107 Bootstrap 36/100
2012-10-13 12:54:10,967       INFO dataset.bootstrap@107 Bootstrap 37/100
2012-10-13 12:54:10,967       INFO dataset.bootstrap@107 Bootstrap 38/100
2012-10-13 12:54:10,967       INFO dataset.bootstrap@107 Bootstrap 39/100
2012-10-13 12:54:10,968       INFO dataset.bootstrap@107 Bootstrap 40/100
2012-10-13 12:54:10,968       INFO dataset.bootstrap@107 Bootstrap 41/100
2012-10-13 12:54:10,968       INFO dataset.bootstrap@107 Bootstrap 42/100
2012-10-13 12:54:10,969       INFO dataset.bootstrap@107 Bootstrap 43/100
2012-10-13 12:54:10,969       INFO dataset.bootstrap@107 Bootstrap 44/100
2012-10-13 12:54:10,969       INFO dataset.bootstrap@107 Bootstrap 45/100
2012-10-13 12:54:10,969       INFO dataset.bootstrap@107 Bootstrap 46/100
2012-10-13 12:54:10,970       INFO dataset.bootstrap@107 Bootstrap 47/100
2012-10-13 12:54:10,970       INFO dataset.bootstrap@107 Bootstrap 48/100
2012-10-13 12:54:10,970       INFO dataset.bootstrap@107 Bootstrap 49/100
2012-10-13 12:54:10,970       INFO dataset.bootstrap@107 Bootstrap 50/100
2012-10-13 12:54:10,971       INFO dataset.bootstrap@107 Bootstrap 51/100
2012-10-13 12:54:10,971       INFO dataset.bootstrap@107 Bootstrap 52/100
2012-10-13 12:54:10,971       INFO dataset.bootstrap@107 Bootstrap 53/100
2012-10-13 12:54:10,972       INFO dataset.bootstrap@107 Bootstrap 54/100
2012-10-13 12:54:10,972       INFO dataset.bootstrap@107 Bootstrap 55/100
2012-10-13 12:54:10,972       INFO dataset.bootstrap@107 Bootstrap 56/100
2012-10-13 12:54:10,972       INFO dataset.bootstrap@107 Bootstrap 57/100
2012-10-13 12:54:10,973       INFO dataset.bootstrap@107 Bootstrap 58/100
2012-10-13 12:54:10,973       INFO dataset.bootstrap@107 Bootstrap 59/100
2012-10-13 12:54:10,973       INFO dataset.bootstrap@107 Bootstrap 60/100
2012-10-13 12:54:10,974       INFO dataset.bootstrap@107 Bootstrap 61/100
2012-10-13 12:54:10,974       INFO dataset.bootstrap@107 Bootstrap 62/100
2012-10-13 12:54:10,974       INFO dataset.bootstrap@107 Bootstrap 63/100
2012-10-13 12:54:10,974       INFO dataset.bootstrap@107 Bootstrap 64/100
2012-10-13 12:54:10,975       INFO dataset.bootstrap@107 Bootstrap 65/100
2012-10-13 12:54:10,975       INFO dataset.bootstrap@107 Bootstrap 66/100
2012-10-13 12:54:10,975       INFO dataset.bootstrap@107 Bootstrap 67/100
2012-10-13 12:54:10,975       INFO dataset.bootstrap@107 Bootstrap 68/100
2012-10-13 12:54:10,976       INFO dataset.bootstrap@107 Bootstrap 69/100
2012-10-13 12:54:10,976       INFO dataset.bootstrap@107 Bootstrap 70/100
2012-10-13 12:54:10,976       INFO dataset.bootstrap@107 Bootstrap 71/100
2012-10-13 12:54:10,977       INFO dataset.bootstrap@107 Bootstrap 72/100
2012-10-13 12:54:10,977       INFO dataset.bootstrap@107 Bootstrap 73/100
2012-10-13 12:54:10,977       INFO dataset.bootstrap@107 Bootstrap 74/100
2012-10-13 12:54:10,977       INFO dataset.bootstrap@107 Bootstrap 75/100
2012-10-13 12:54:10,978       INFO dataset.bootstrap@107 Bootstrap 76/100
2012-10-13 12:54:10,978       INFO dataset.bootstrap@107 Bootstrap 77/100
2012-10-13 12:54:10,978       INFO dataset.bootstrap@107 Bootstrap 78/100
2012-10-13 12:54:10,978       INFO dataset.bootstrap@107 Bootstrap 79/100
2012-10-13 12:54:10,979       INFO dataset.bootstrap@107 Bootstrap 80/100
2012-10-13 12:54:10,979       INFO dataset.bootstrap@107 Bootstrap 81/100
2012-10-13 12:54:10,979       INFO dataset.bootstrap@107 Bootstrap 82/100
2012-10-13 12:54:10,980       INFO dataset.bootstrap@107 Bootstrap 83/100
2012-10-13 12:54:10,980       INFO dataset.bootstrap@107 Bootstrap 84/100
2012-10-13 12:54:10,980       INFO dataset.bootstrap@107 Bootstrap 85/100
2012-10-13 12:54:10,980       INFO dataset.bootstrap@107 Bootstrap 86/100
2012-10-13 12:54:10,981       INFO dataset.bootstrap@107 Bootstrap 87/100
2012-10-13 12:54:10,981       INFO dataset.bootstrap@107 Bootstrap 88/100
2012-10-13 12:54:10,981       INFO dataset.bootstrap@107 Bootstrap 89/100
2012-10-13 12:54:10,981       INFO dataset.bootstrap@107 Bootstrap 90/100
2012-10-13 12:54:10,982       INFO dataset.bootstrap@107 Bootstrap 91/100
2012-10-13 12:54:10,982       INFO dataset.bootstrap@107 Bootstrap 92/100
2012-10-13 12:54:10,982       INFO dataset.bootstrap@107 Bootstrap 93/100
2012-10-13 12:54:10,983       INFO dataset.bootstrap@107 Bootstrap 94/100
2012-10-13 12:54:10,983       INFO dataset.bootstrap@107 Bootstrap 95/100
2012-10-13 12:54:10,983       INFO dataset.bootstrap@107 Bootstrap 96/100
2012-10-13 12:54:10,983       INFO dataset.bootstrap@107 Bootstrap 97/100
2012-10-13 12:54:10,984       INFO dataset.bootstrap@107 Bootstrap 98/100
2012-10-13 12:54:10,984       INFO dataset.bootstrap@107 Bootstrap 99/100
2012-10-13 12:54:10,984       INFO halla._halla_test@82 Testing 0/2
2012-10-13 12:54:10,985       INFO halla._halla_test@82 Testing 1/2
