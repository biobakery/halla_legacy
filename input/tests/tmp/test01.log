2012-10-13 12:22:00,360       INFO dataset.permute@90 Generating 41 permutations
2012-10-13 12:22:00,360       INFO dataset.permute@94 Permutation 0/41
2012-10-13 12:22:00,361       INFO dataset.permute@94 Permutation 1/41
2012-10-13 12:22:00,361       INFO dataset.permute@94 Permutation 2/41
2012-10-13 12:22:00,361       INFO dataset.permute@94 Permutation 3/41
2012-10-13 12:22:00,362       INFO dataset.permute@94 Permutation 4/41
2012-10-13 12:22:00,362       INFO dataset.permute@94 Permutation 5/41
2012-10-13 12:22:00,362       INFO dataset.permute@94 Permutation 6/41
2012-10-13 12:22:00,362       INFO dataset.permute@94 Permutation 7/41
2012-10-13 12:22:00,363       INFO dataset.permute@94 Permutation 8/41
2012-10-13 12:22:00,363       INFO dataset.permute@94 Permutation 9/41
2012-10-13 12:22:00,363       INFO dataset.permute@94 Permutation 10/41
2012-10-13 12:22:00,364       INFO dataset.permute@94 Permutation 11/41
2012-10-13 12:22:00,364       INFO dataset.permute@94 Permutation 12/41
2012-10-13 12:22:00,364       INFO dataset.permute@94 Permutation 13/41
2012-10-13 12:22:00,365       INFO dataset.permute@94 Permutation 14/41
2012-10-13 12:22:00,365       INFO dataset.permute@94 Permutation 15/41
2012-10-13 12:22:00,365       INFO dataset.permute@94 Permutation 16/41
2012-10-13 12:22:00,366       INFO dataset.permute@94 Permutation 17/41
2012-10-13 12:22:00,366       INFO dataset.permute@94 Permutation 18/41
2012-10-13 12:22:00,366       INFO dataset.permute@94 Permutation 19/41
2012-10-13 12:22:00,367       INFO dataset.permute@94 Permutation 20/41
2012-10-13 12:22:00,367       INFO dataset.permute@94 Permutation 21/41
2012-10-13 12:22:00,367       INFO dataset.permute@94 Permutation 22/41
2012-10-13 12:22:00,367       INFO dataset.permute@94 Permutation 23/41
2012-10-13 12:22:00,368       INFO dataset.permute@94 Permutation 24/41
2012-10-13 12:22:00,368       INFO dataset.permute@94 Permutation 25/41
2012-10-13 12:22:00,368       INFO dataset.permute@94 Permutation 26/41
2012-10-13 12:22:00,369       INFO dataset.permute@94 Permutation 27/41
2012-10-13 12:22:00,369       INFO dataset.permute@94 Permutation 28/41
2012-10-13 12:22:00,369       INFO dataset.permute@94 Permutation 29/41
2012-10-13 12:22:00,370       INFO dataset.permute@94 Permutation 30/41
2012-10-13 12:22:00,370       INFO dataset.permute@94 Permutation 31/41
2012-10-13 12:22:00,370       INFO dataset.permute@94 Permutation 32/41
2012-10-13 12:22:00,371       INFO dataset.permute@94 Permutation 33/41
2012-10-13 12:22:00,371       INFO dataset.permute@94 Permutation 34/41
2012-10-13 12:22:00,371       INFO dataset.permute@94 Permutation 35/41
2012-10-13 12:22:00,371       INFO dataset.permute@94 Permutation 36/41
2012-10-13 12:22:00,372       INFO dataset.permute@94 Permutation 37/41
2012-10-13 12:22:00,372       INFO dataset.permute@94 Permutation 38/41
2012-10-13 12:22:00,372       INFO dataset.permute@94 Permutation 39/41
2012-10-13 12:22:00,373       INFO dataset.permute@94 Permutation 40/41
2012-10-13 12:22:00,373       INFO dataset.hierarchy@205 Cluster 1 (3): ['D1', 'D3', 'D4']
2012-10-13 12:22:00,373       INFO dataset.bootstrap@104 Generating 100 bootstraps
2012-10-13 12:22:00,373       INFO dataset.bootstrap@107 Bootstrap 0/100
Traceback (most recent call last):
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/halla.py", line 153, in <module>
    _main( )
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/halla.py", line 150, in _main
    halla( args.istm, args.ostm, args.dP, args.dPMI, args.iBootstrap, args.iUnique )
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/halla.py", line 109, in halla
    _halla_test( ostm, pData, hashClusters, dP, iBootstrap )
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/halla.py", line 51, in _halla_test
    pData.bootstrap( iBootstrap )
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/dataset.py", line 113, in bootstrap
    self.m_aadBootstraps.append( self._pdist( self.m_apData, hashBootstrap ) )
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/dataset.py", line 61, in _pdist
    aCache = [pDatum.cache( hashIndices ) for pDatum in apData]
  File "/n/home10/chuttenhower/hg/sfle/input/halla/src/datum.py", line 248, in cache
    setiIndices = set(hashIndices.keys( ))
AttributeError: 'list' object has no attribute 'keys'
