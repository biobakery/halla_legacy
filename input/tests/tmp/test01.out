Clusters
D1	D3	D4
